architecture/core_concepts/routes.adoc

```diff
@@ -185,22 +185,27 @@ example, for a cluster with five back-end pods, and two load-balanced routers,
 you can ensure that the same pod receives the web traffic from the same web
 browser regardless of the router that handles it.
 
-While placing returning traffic onto the same pod cannot be guaranteed, you can
+While routing returning traffic onto the same pod cannot be guaranteed, you can
 use HTTP headers to set a cookie determining the pod used by the last
 connection, in an attempt to steer traffic to the same pod. When the application
-user hits the router the browser re-sends the cookie and knows where to send the
-traffic.
+user hits the router the browser re-sends the cookie and the router knows where
+to send the traffic.
 
 As a cluster administrator, you can turn off the stickiness for passthrough
 routes separately from other connections, or turn off stickiness entirely.
 
 By default, if the default HAProxy router is using passthrough routes, sticky
-sessions are implemented using the `balance source` directive in the underlying
-router configuration, which balances based on the source IP. Other types of
-routes use `balance leastconn` by default. If a cookie is present, it is read
-and sent to the same pod it went to last time if the pod still exists. If no
-cookie is present, because this is a new session, stickiness is disabled, or
-it's using a passthrough route, the router picks a pod.
+sessions are implemented using the `source` xref:load-balancing[load balancing strategy].
+This can be changed for a route using a xref:route-specific-annotations[route annotation] or
+for all pass through routes by using the `*ROUTER_TCP_BALANCE_SCHEME*`
+xref:env-variables[environment variable]. The `source` balancing strategy
+balances based on the source IP. Other types of routes use `leastconn` by
+default which can be changed by using the `*ROUTER_LOAD_BALANCE_ALGORITHM*`
```

Comment:
You perhaps want to mention that it can be set per-route by an annotation, or restructure the way you talk about the defaults so the passthrough case isn't joined by a comma to the annotations, but the other case is a new sentence.